tasks/sonar_runner.js

```diff
@@ -44,7 +44,10 @@ module.exports = function (grunt) {
         var dryRun = options.dryRun;
         var done = this.async();
 
-        options.sonar.language = options.sonar.language || 'js';
```

Comment:
ok. sure. if not putting a default. we will need more documentation on sonar.language option to say not required for multi. required if doing only one language analysis